web/src/stream_edit.js

```diff
@@ -415,18 +414,6 @@ export function get_stream_email_address(flags, address) {
 }
 
 export function initialize() {
-    $("#main_div").on("click", ".stream_sub_unsub_button", (e) => {
-        e.preventDefault();
-        e.stopPropagation();
-
-        const sub = narrow_state.stream_sub();
-        if (sub === undefined) {
-            return;
-        }
-
-        stream_settings_ui.sub_or_unsub(sub);
-    });
-
```

Comment:
I'm confused; this click handler seems to be for the notices in the message feed offering to subscribe to a stream; where is the other click handler for this button?

```
web/templates/bookend.hbs:        <button class="button white rounded stream_sub_unsub_button {{#unless subscribed}}sea-green{{/unless}}" type="button" name="subscription">
```

Reply:
Ohh woops my bad. Thanks :)